app/src/main/java/org/p2p/wallet/home/ui/container/MainContainerFragment.kt

```diff
@@ -184,7 +182,7 @@ class MainContainerFragment :
     override fun navigateFromDeeplink(data: DeeplinkData) {
         when (data.target) {
             DeeplinkTarget.HOME -> {
-                navigate(ScreenTab.HOME_SCREEN)
+                navigate(ScreenTab.MY_CRYPTO_SCREEN)
```

Comment:
maybe we should rename for clarity the DeeplinkTarget.HOME to DeeplinkTarget.MY_CRYPTO ?